packages/toasts/src/index.js

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
nit: Is this eslint comment still relevant with multiple exports in this file now?